metpetdb_api/api/chemical_analyses/lib/query.py

```diff
@@ -8,6 +8,12 @@ def chemical_analysis_query(user, params, qs):
     else:
         qs = qs.filter(Q(owner=user) | Q(public_data=True))
 
+    if params.get('provenance'):
+        if(params['provenance']=="Public"):
```

Comment:
Extra parenthesis in Python are superfluous. Statements of the form 
`if(params['provenance']=="Public")`
should be writen
`if params['provenance']=="Public"`

Noted here, but all should be changed for consistency.